data/json/recipes/recipe_deconstruction.json

```diff
@@ -1158,6 +1158,183 @@
       [ [ "storage_battery", 1 ] ]
     ]
   },
+  {
+    "result": "broken_exodii_worker",
+    "type": "uncraft",
+    "skill_used": "electronics",
+    "skills_required": [ [ "mechanics", 9 ], [ "computer", 9 ] ],
```

Comment:
Why do you need computer skill for these?

Reply:
It's a general substitute for the fact that if you're going to uncraft it rather than just tear it apart, you're going to need to know a great deal about a great many things. Fluff-wise, exodii parts lock up if incorrectly rebooted, to prevent zombies from using them. You'd need to know enough about computers to hack past that, or you'd get little more than scrap at the end.